Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.10.4-alpine
+LABEL maintainer="alexzubkoff123@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+ENV API_KEY=8b89884a56104151964200013222711
```

Comment:
Pushing sensitive data such as this key to the source control is generally not the best idea. It might not be important here (and it might even be the intended solution for the task), but I'd suggest leaving your key secret and only creating the variable locally.